LandingPage.js

```diff
@@ -0,0 +1,69 @@
+import styles from './LandingPage.module.css';
+import classNames from 'classnames';

```

Comment:
not sure what this import is, but it seems like its not being used so you can remove this